Library/ViewModels/DiscoveryFiltersViewModelTests.swift

```diff
@@ -44,7 +44,7 @@ private let filmExpandableRow = expandableRowTemplate
 private let categories = [ Category.art, .illustration, .filmAndVideo, .documentary ]
 
 internal final class DiscoveryFiltersViewModelTests: TestCase {
-  private let vm = DiscoveryFiltersViewModel()
+  private var vm: DiscoveryFiltersViewModelType = DiscoveryFiltersViewModel()
```

Comment:
Ah, I changed it to `var` because I moved the instantiation to `setUp()` but forgot to change it back to `let`, thanks!